code/game/objects/items/weapons/dna_injector.dm

```diff
@@ -43,7 +43,7 @@
 	if(buf.types & DNA2_BUF_SE)
 		return buf.dna.GetSEState(real_block)
 	else
-		return buf.dna.GetUIState(real_block)
+		return buf.dna.GetUITriState(real_block)
```

Comment:
This will break a ton of the other UI data

Reply:
**_HORF_**.

Reply:
Somehow, someway, someone (me) thought this was related to gender.
👏 nice job me 👏